web/public/templates/dojos/events/apply.dust

```diff
@@ -18,7 +18,7 @@
       <p>{{ event.city.nameWithHierarchy }}</p>
     </div>
     <div class="col-md-6">
-      <p>{{ event.description }}</p>
+      <p><div ng-bind-html="event.description"></div></p>
```

Comment:
We don't need to add an extra `div` element here, we can just do `<p ng-bind-html="event.decsription">`. Unless you really need it to be a block element (in which case you could have changed the `p` to a `div`.